recipes/autoconf/all/test_v1_package/conanfile.py

```diff
@@ -0,0 +1,47 @@
+from conans import AutoToolsBuildEnvironment, ConanFile, tools
+from conan.tools.microsoft import is_msvc, unix_path
+import contextlib
+import os
+import shutil
+
+required_conan_version = ">=1.45.0"
+
+
+class TestPackageConan(ConanFile):
+    settings = "os", "compiler", "build_type", "arch"
+    exports_sources = "configure.ac", "config.h.in", "Makefile.in", "test_package_c.c", "test_package_cpp.cpp",
```

Comment:
Do these files need to be duplicated or can we pull them copy them from the other test folder?

Reply:
Should be possible